src/index.native.tsx

```diff
@@ -47,6 +47,8 @@ function enableScreens(shouldEnableScreens = true): void {
 
 // const that tells if the library should use new implementation, will be undefined for older versions
 const shouldUseActivityState = true;
+const isNavigationContainerAvailable =
```

Comment:
Is there a case when this won't be available when someone is using this version of react-native-screens?

Reply:
What if someone updates to this version of library in Expo managed workflow? I know it should not be done, but 🤷‍♂️ .

Reply:
@WoLewicki updating to an incompatible version of the library could also cause other issues as well, and surface bugs which are already fixed in latest version, so probably not a use case we should support.